src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const expensesService = require('../services/expenses.service');
+const userService = require('../services/users.service');
+
+const getAll = async(req, res) => {
+  const {
+    userId,
+    from,
+    to,
+    categories,
+  } = req.query;
+
+  const expenses = await expensesService.getAll({
+    userId,
+    from,
+    to,
+    categories,
+  });
+
+  res.send(
+    expenses.map(expensesService.normalize)
+  );
+};
+
+const getOne = async(req, res) => {
+  const { expenseId } = req.params;
```

Comment:
SAme about the required data here and in all other cases in an expenses controller